src/routes/+page.svelte

```diff
@@ -131,7 +131,12 @@
     const showCollectionsOnFirstLaunch = config.mainFeatures['layout-config-first-launch'];
     const showCollectionViewer = config.mainFeatures['layout-config-change-viewer-button'];
     const showAudio = config.mainFeatures['audio-allow-turn-on-off'];
-    $: showBorder = config.traits['has-borders'] && $userSettings['show-border'];
+    $: showBorderSetting = getFeatureValueString(
```

Comment:
Why use `getFeatureValueString` instead `getFeatureValueBoolean`?  It seems that the values are `true`, `false`, or `inherited`.